charts/airbyte/templates/temporal/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       {{- end }}
       containers:
       - name: airbyte-temporal
-        image: "{{ .Values.temporal.image.repository}}:{{ .Values.temporal.image.tag }}"
+        image: {{ include "airbyte.temporalImage" . }}
```

Comment:
This was something I missed in the last PR. It makes the `global.imagePullPolicy` work for the temporal image.